app/proposal/[index].tsx

```
@@ -0,0 +1,306 @@
+        height={"100%"}
+        rx={"2.5"}
+        ry={"2.5"}
+        fill={colorNay}
+      />
+    </Svg>
+  );
+}
+
+interface SeeDetailsProps {}
+
+function SeeDetails({}: SeeDetailsProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)